sudo.ps1

```diff
@@ -61,9 +66,10 @@ $wd = serialize (convert-path $pwd) # convert-path in case pwd is a PSDrive
 
 $savetitle = $host.ui.rawui.windowtitle
 $p = new-object diagnostics.process; $start = $p.startinfo
-$start.filename = "powershell.exe"
-$start.arguments = "-noprofile & '$pscommandpath' -do $wd $pid $a`nexit `$lastexitcode"
+$start.filename = "$powershellExe"
+$start.arguments = "-noprofile -Command & '$pscommandpath' -do $wd $pid $a`nexit `$lastexitcode"
```

Comment:
Same as said in l. 31:

    $start.arguments = "-noprofile $commandPrefix & '$pscommandpath' -do $wd $pid $a`nexit `$lastexitcode"